app/views/moderations/mod.html.erb

```diff
@@ -1,127 +1,10 @@
-<style>
-  .reaction-button {
-    position: relative;
-    width: 60px;
-    height: 60px;
-    background: white;
-    border-color: white;
-    margin: 100px 40px;
-    border-radius: 200px;
-  }
+<% title "[Moderate] " + @moderatable.title %>
 
-  .reaction-button img {
-    position: absolute;
-    top: 0;
-    width: 60px;
-    left: 0;
-  }
-
-  .reaction-button .reacted-emoji {
-    display: none;
-  }
-
-  .reaction-button.reacted .reacted-emoji {
-    display: block;
-  }
-
-  .tag-mod-form {
-    border-top: 2px solid #dddddd;
-    margin: 40px auto;
-  }
-
-  .tag-mod-form ul {
-    text-align: left;
-    font-size: 15px;
-  }
-
-  .tag-mod-form form {
-    width: 500px;
-    margin: auto;
-    max-width: 90%;
-  }
-
-  .tag-mod-form form input {
-    width: 100%;
-    padding: 5px;
-    font-size: 20px;
-    margin-bottom: 5px;
-    border: 1px solid #888;
-  }
-
-  .tag-mod-form form input[type="submit"] {
-    background: #0045ff;
-    color: white;
-    font-weight: bold;
-    padding: 10px;
-    margin: auto;
-    display: block;
-  }
-
-  #undo {
-    background: #ff0000;
-    width: 2rem;
-    height: 2rem;
-    font-size: small;
-    display: inline-block;
-    padding-bottom: 2em;
-  }
-
-  .tag-mod-form form textarea {
-    box-sizing: border-box;
-    width: 100%;
-    padding: 5px;
-    font-size: 17px;
-    height: 100px;
-    border: 1px solid #888;
-  }
-
-  .level-rating-button {
-    border-radius: 8px;
-    font-size: 1.1em;
-    padding: 3px;
-    width: 42px;
-    border: 2px solid black;
-    font-weight: bold;
-  }
-
-  .level-rating-button.selected {
-    background: #9bebff;
-  }
-
-  input.undo {
-    background: #ff0000; /* $red */
-  }
-
-  .tag-mod-form input[type=radio] {
-    position: absolute;
-  }
-
-  .tag-mod-form input[type=radio] + label {
-    display: inline-block;
-    cursor: pointer;
-    font-weight: bold;
-    padding-left: 25px;
-    padding-right: 25px;
-    font-size: 20px;
-    margin: auto;
-    margin-bottom: 15px;
-  }
-
-  .tag-mod-form input[type=radio]:checked + label{
-    color: white;
-    background: #0045ff;
-  }
-
-  .tag-mod-form select {
-    display: block;
-    font-size: 20px;
-    padding: 12px;
-    font-weight: bold;
-    margin: auto;
-    border: solid 2px;
-    margin-top: 5px;
-  }
-</style>
+<% cache "mod-styling-#{ApplicationConfig['HEROKU_SLUG_COMMIT']}" do %>
```

Comment:
Are there more places we can do this? If so, we could drop this into a partial, probably.